formats/hmrc_manual/publisher/links.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "topics": {
+      "anyOf": [
+        { "$ref": "#/definitions/guid_list" },
+        { "type": "null" }
+      ]
```

Comment:
I don't think this `anyOf` stuff is necessary.  These fields aren't required by default (`requiredProperties` isn't set on the `links` object by default.)